redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
I have changed both `time-source` and `log-sink`, e.g.: https://github.com/spring-cloud/spring-cloud-stream-modules/commit/67bea7588a6802e2658ef86db65e9a457c11fdd9